messages/src/lib.rs

```diff
@@ -1961,6 +1961,133 @@ impl Decode for AggregateShareResp {
     }
 }
 
+/// Representation of the different problem types defined in Table 1 in §3.2.
+#[derive(Debug, PartialEq, Eq)]
+pub enum DapProblemType {
```

Comment:
Could we move this to a new file like `messages/src/problem_type.rs` just to stop this file from getting too big? It can be a little frustrating to navigate `janus` when you have 4-5 different files named `lib.rs` open and they're all very long.